### DISCUSSION TITLE
an error with bootcamp

### DISCUSSION THREAD

**Discussion by mfscool** [osy86/HaC-Mini#996] [General]
I got an error when installing windows 10 with bootcamp.

![image](https://user-images.githubusercontent.com/43715026/160758464-20d03417-3858-4bea-8132-909f487f33fa.png)

**Comment by mfscool** [comment]
downloading windows support software……

Can’t install the software because it is not currently available from the Software Update server.

**Comment by xiaooloong** [comment]
推荐还是用另外一块硬盘直接装系统，开机 F10 切换。
在 Windows 下可以用 BootICE1.3.4.0 这个绿色软件调整 UEFI 的顺序。不要用 eazyUEFI，有 BIOS 砖掉的风险。

另外别说 BootCamp 了，就是直接装的 Windows，用 OpenCore 引导进 Windows 硬件信息也被打了 Mac 的补丁，频繁需要重新登录微软账号。

**Comment by mfscool** [comment]
谢谢指导，我弄个硬盘按照你的方法试一下。
方便加个微信吗？




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年3月30日(星期三) 下午2:20
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [osy/HaC-Mini] an error with bootcamp (Issue #993)





 
推荐还是用另外一块硬盘直接装系统，开机 F10 切换。
 在 Windows 下可以用 BootICE1.3.4.0 这个绿色软件调整 UEFI 的顺序。不要用 eazyUEFI，有 BIOS 砖掉的风险。
 
另外别说 BootCamp 了，就是直接装的 Windows，用 OpenCore 引导进 Windows 硬件信息也被打了 Mac 的补丁，频繁需要重新登录微软账号。

**Comment by mfscool** [comment]
> 还是推荐用另外的硬盘直接安装系统，引导F10切换。 在Windows下可以用BootICE1.3.4.0这个软件调整UEFI的顺序。不要用eazyUEFI，有BIOS砖掉的风险。
> 
> 另外别说 BootCamp 的 Windows，用 OpenCore 进出 Windows 硬件信息也被 Mac 的位置，直接直接登录了微软的日志。

您好，有相关的教程吗？我的理解是U盘启动，两个硬盘分别装两个系统，然后f10开机时选取由哪个硬盘启动，然后bootice不太清楚作用是什么，麻烦指导一下，谢谢。

**Comment by Medizinmann-MD** [comment]
One question...

Why bootcamp?

I mean - just pop in a 2nd SSD and boot from there - this is like I do it - no need to fiddle around.

**Comment by mfscool** [comment]
> One question...
> 
> Why bootcamp?
> 
> I mean - just pop in a 2nd SSD and boot from there - this is like I do it - no need to fiddle around.

Thanks

**Comment by Amoylee49** [comment]
> > 还是推荐用另外的硬盘直接安装系统，引导F10切换。 在Windows下可以用BootICE1.3.4.0这个软件调整UEFI的顺序。不要用eazyUEFI，有BIOS砖掉的风险。
> > 另外别说 BootCamp 的 Windows，用 OpenCore 进出 Windows 硬件信息也被 Mac 的位置，直接直接登录了微软的日志。
> 
> 您好，有相关的教程吗？我的理解是U盘启动，两个硬盘分别装两个系统，然后f10开机时选取由哪个硬盘启动，然后bootice不太清楚作用是什么，麻烦指导一下，谢谢。

https://shaoxing.netlify.app/6629381b.html
我是在mac系统下挂载，然后开机按Alt可以选择哪个启动，mac下也可以设置先启动哪个盘，两个盘独立系统

**Comment by Medizinmann-MD** [comment]
If you have a 2nd machine with Windows - you could use an external enclosure and WinToUSB - IMHO the easiest way:
https://www.easyuefi.com/wintousb/
Then install the drive in the 2nd M.2-Slot - and it should show up in the OpenCore Boot Picker - you then are able to choose between booting MacOS or Windows.

If no other Windows machine is in reach you can follow this guide...
https://www.freecodecamp.org/news/how-make-a-windows-10-usb-using-your-mac-build-a-bootable-iso-from-your-macs-terminal/

When I first installed Windows I pulled the drive with MacOS out beforehand - just in case - sometimes Windows-Installer overwrites Bootdrives.
